doc/lua_api.txt

```diff
@@ -932,95 +918,94 @@ use them to store arbitrary values.
 Node paramtypes
 ---------------
 The functions of `param1` and `param2` are determined by certain fields in the
-node definition:
+node definition.
 
 `param1` is reserved for the engine when `paramtype != "none"`:
 
-    paramtype = "light"
-    ^ The value stores light with and without sun in its upper and lower 4 bits
+* `paramtype = "light"`
+    * The value stores light with and without sun in its upper and lower 4 bits
       respectively.
-      Required by a light source node to enable spreading its light.
-      Required by the following drawtypes as they determine their visual
+    * Required by a light source node to enable spreading its light.
+    * Required by the following drawtypes as they determine their visual
       brightness from their internal light value:
-        torchlike,
-        signlike,
-        firelike,
-        fencelike,
-        raillike,
-        nodebox,
-        mesh,
-        plantlike,
-        plantlike_rooted.
+        * torchlike
```

Comment:
The two extra spaces are of no use now; the `*` can align with the start of the previous line like so:

```
* Required by the following drawtypes ...
  brightness from their ...
  * torchlike
  * signlike
```

Reply:
4 spaces are used everywhere else for nested lists and it's required by some markdown implementations